### DISCUSSION TITLE
Add more pen-button options

### DISCUSSION THREAD

**Discussion by issamugael** [OpenTabletDriver/OpenTabletDriver#782] [Ideas]
I think that will be good if the driver had more pen button options, like zoom in and out functions, open up program with a unique click and combined keys, like Alt + F4

What do you guys think?

**Comment by InfinityGhost** [comment]
This is planned, see #740

**Comment by hbayard** [comment]
> I think that will be good if the driver had more pen button options, like zoom in and out functions, open up program with a unique click and combined keys, like Alt + F4
> 
> What do you guys think?

+1 to this — double click! Been losing my mind trying to figure out if I was missing something
